pupa/scrape/base.py

```diff
@@ -5,9 +5,10 @@
 from collections import defaultdict, OrderedDict
 
 import scrapelib
+from django.conf import settings
```

Comment:
Sorry, it's not clear what this changes. We are using pupa_settings.py now our scrapers-municipal directory.